src/styles/partials/components/_alt-list.scss

```diff
@@ -0,0 +1,56 @@
+ol.dg_alt-list {
+  list-style-type: none;
+  margin-bottom: $margin-large;
+  margin-left: 0;
+  padding-left: 0;
+  text-decoration: none;
+}
+
+ul.dg_alt-list {
+  list-style-type: none;
+  margin-bottom: $margin-large;
+  margin-left: 0;
+  padding-left: 0;
+  text-decoration: none;
+}
+
+.dg_i-float-right {
```

Comment:
once changed to use will be positioned rather than floated.

Reply:
Working on this as well, since it goes hand-in-hand with the previous update.